src/components/Header.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  import logo from '/src/assets/svelte.svg';
+  import { Button } from 'flowbite-svelte';
+
+  const dummyCity = 'Ukraine, Kyiv, elevation 199m';
+  const dummyIsSearch = false;
```

Comment:
Maybe 'isSearchContext'? Or I am missing something?

Reply:
We will merge this PR after this one, so I remove this variable